src/System.Collections/tests/Generic/HashSet/HashSetTestSupport.cs

```diff
@@ -1015,7 +1016,7 @@ public class Item
         {
             public int x;
 
-            private static int s_nextIndex = 1;
+            private static int s_nextIndex = 0;
             private static object s_nextIndexLock = new object();
```

Comment:
No, we do not.  

I'm not sure what lead to me discovering and making that particular change in the first place.  Maybe I thought I was getting deadlock or something?  Still, use of interlocked reads cleaner...
There a second instance of this in the file, so replacing that too.